ci-operator/config/3scale-qe/3scale-deploy/3scale-qe-3scale-deploy-main__3scale-amp-ocp4.14-lp-rosa.yaml

```diff
@@ -0,0 +1,75 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  test-image:
+    name: 3scale-interop-tests
+    namespace: ci
+    tag: v2.13
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: 3scale-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: 3scale-amp-interop-rosa-aws
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      AWS_REGION: us-east-2
+      DEPL_BUCKET_NAME: 3scale-apimanager-s3-bucket
+      DEPL_PROJECT_NAME: threescale
+      FIREWATCH_CONFIG: |
```

Comment:
Can we use the firewatch v2 syntax for this firewatch config?

This will allow us to specify jira_additonal_labels in a single place instead of in every line among some other things.

Here's Caleb PR where he introduced the new firewatch syntax to our self-managed scenarios
https://github.com/openshift/release/pull/46282